sympy/functions/elementary/complexes.py

```diff
@@ -443,6 +449,20 @@ def eval(cls, arg):
             return known*unk
         if arg is S.NaN:
             return S.NaN
+        if arg.is_Pow:
+            base, exponent = arg.as_base_exp()
+            if base.is_real:
+                if exponent.is_integer:
+                    if exponent.is_even:
+                        return arg
+                    if base is S.NegativeOne:
+                        return S.One
```

Comment:
``base`` is ``NegativeOne`` and ``exponent`` is odd - therefore this function should return ``NegativeOne`` instead of ``One``